recipes/bursth2mm/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "burstH2MM" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/harripd/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: d06654d8e37a1d30ce39c97f861bcc1542b25333fecf43ae5c1a07a2e1892a8f
+
+build:
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
As long as it still builds

Reply:
@harripd, I'm a conda-forge core member and staged-recipes reviewer. Why do you think I would send you something that would not work?

Reply:
I didn't see any indication that it couldn't build, I wasn't saying I did trust you, sorry if I implied that.